src/Umbraco.Web/Editors/ContentController.cs

```diff
@@ -895,7 +895,7 @@ public HttpResponseMessage PostMove(MoveOrCopy move)
         {
             var toMove = ValidateMoveOrCopy(move);
 
-            Services.ContentService.Move(toMove, move.ParentId);
+            Services.ContentService.Move(toMove, move.ParentId, Security.GetUserId());

```

Comment:
Might be better to use `Security.CurrentUser.Id`. I see there's some more usages of `GetUserId()` in the rest of the code that could also just use `Security.CurrentUser.Id`.

Reply:
@nul800sebastiaan sure no problem, I'll replace all of 'em while I'm at it.